### ISSUE TITLE
What were the hyperparameters for your paper AEN for Targeted Sentiment Classification?

### ISSUE THREAD

**Issue by momofosho** [songyouwei/ABSA-PyTorch#142]
Hello, I'm an undergraduate trying to run the code from your paper "Attentional Encoder Network for Targeted Sentiment Classification", thank you so much for your work, but I am having trouble getting the same results, may I confirm that the hyperparameters are embed_dim = 300, bert_dim = 768, hidden_dim = 300, initializer = 'xavier_normal_' (or is it uniform?), l2reg = e-5, dropout = 0.1, optimizer = 'adam'? Did I miss out some things? Thank you, in advance, for your help!

**Comment by songyouwei** [comment]
Please refer to issue #62

**Comment by momofosho** [comment]
Hello, thank you so much for your help, but I am still running into problems running the code. I tried running train k-fold cross validation on AEN-BERT, but somehow the variables context=self.dropout and target=self.dropout in AEN-BERT returns a list of 12 tensors, instead of 1, leading to a type error. It works ok with just train.py though, so I am not sure what to change if I want to use train_k_fold_cross_val.py. Sorry, do you have more advice on how to replicate your results from the paper? Thank you.
